res/css/views/rooms/_EventTile.pcss

```diff
@@ -1032,20 +1032,14 @@ $left-gutter: 64px;
     --ThreadView_group_spacing-end: 8px; /* same as padding */
 
     .mx_EventTile {
-        display: flex;
```

Comment:
https://github.com/matrix-org/matrix-react-sdk/pull/10335/files#diff-5d3e25eb1d22dd7a96e0e87ea9c1298dc7dbce8eb965d30b2d1ebc1436bfec3fR1105

Applied to bubble layout only